neuromation/api/utils.py

```diff
@@ -108,3 +109,7 @@ def retries(
         sleeptime += 0.1
         attempts -= 1
         yield retry()
+
+
+def _delta_total_minutes(delta: timedelta) -> int:
+    return int(delta.total_seconds() // 60)
```

Comment:
should this be in `neuromation/api/jobs.py`, where it is used?

Reply:
good idea